oaknational/Oak-Web-Application
# Added grouping via subject categories for english subject

## Description
Subject category grouping for english primary in curriculum downloads (docx) units/threads pages   

## Issue(s)

Fixes

 - `CUR-829`
 - `CUR-1141`

## How to test

1. Go to https://deploy-preview-3048--oak-web-application.netlify.thenational.academy/teachers/curriculum/
2. Download documents for english and non english curriculum document. 
3. Check that they are grouped by subject category in english and not elsewhere (find particulars, units in `CUR-829` & threads in `CUR-1141`)
